yt/python/yt/wrapper/client_impl.py

```diff
@@ -2496,6 +2496,36 @@ def run_operation(
             client=self,
             sync=sync, run_operation_mutation_id=run_operation_mutation_id, enable_optimizations=enable_optimizations)
 
+    def run_query(
+            self,
+            engine, query,
+            settings=None, files=None, stage=None, annotations=None, access_control_objects=None,
+            sync=True):
+        """
+        Run query and track its progress (unless sync = false).
+
+        :param engine: one of "ql", "yql", "chyt", "spyt".
+        :type engine: str
+        :param query: text of a query
+        :type query: str
+        :param settings: a dictionary of settings
```

Comment:
"settings" is similar to "spec" in case of operation -- it is also an opaque (on a level of SDK) map of various options that are interpreted along the path of query execution (some on QT level, some on a level of a concrete engine like YQL or CHYT).

In any case, I used exactly the same wording as in start_query. I would like not to add significant rewordings to this PR, as it is already quite massive. Better wording can be introdiuced for start_query and run_query simultaneously in the future PRs.